sources/packages/web/src/services/http/dto/Assessment.dto.ts

```diff
@@ -83,3 +83,7 @@ export interface AwardDetailsAPIOutDTO {
    */
   finalAward?: Record<string, string | number>;
 }
+
+export interface ManualReassessmentAPIInDTO {
```

Comment:
The DTO file must be aligned with the API file.
API DTO is currently under application.dto.ts on the API.